inventory/group_vars/br.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Brazil
+country_code: BR
+currency: BRL
+locale: pt-BR                  # Default locale for OFN instance
+available_locales: br, pt    # Available locales for OFN instance
```

Comment:
this would be pt_BR
add pt only if you want to have the language toggle with the option to switch between Portuguese from Brazil and Portuguese from Portugal.